packages/@coorpacademy-components/src/atom/button-link/index.tsx

```diff
@@ -2,13 +2,46 @@ import React, {useCallback} from 'react';
 import {noop} from 'lodash/fp';
 import classnames from 'classnames';
 import Link from '../link';
+import FaIcon from '../icon';
 import {ICONS} from '../../util/button-icons';
 import propTypes, {ButtonLinkProps, IconType} from './types';
 import style from './style.css';
 
 const getButtonContent = (icon?: IconType, label?: string) => {
-  const {type, position} = icon || {type: '', position: ''};
+  const {type, faIconName, faIconColor, faIconBackgroundColor, faIconSize, position, customStyle} =
+    icon || {type: '', position: ''};
   const Icon = type && ICONS[type];
+  const FaIconParsed: any = FaIcon;
```

Comment:
check my comment in packages/@coorpacademy-components/src/atom/button-link/types.ts: 
using PropTypes.InferProps<typeof FaIcon.propTypes>; you won't have to use any, you can directly do this: 
```suggestion
  if (faIconProps) {
    return (
      <div className={style.buttonContent}>
        {position === 'left' ? <FaIcon {...faIconProps} /> : null}
        {label ? <span className={style.label}>{label}</span> : null}
        {position === 'right' ? <FaIcon {...faIconProps} /> : null}
      </div>
    );
  }
```

Reply:
:link: https://github.com/CoorpAcademy/components/pull/2846/files#r1574599333